components/ZilPay/zilpay-base.ts

```diff
@@ -5,336 +5,336 @@ import { toast } from 'react-toastify'
 import { updateShowZilpay } from '../../src/store/modal'
 
 type Params = {
-  contractAddress: string
-  transition: string
```

Comment:
do we have a new config for prettier? this file shouldn't have changed @devwistkey @sherly09

Reply:
Hi @tralkan, I don't have any changes for prettier. What version are you using? Mine v2.6.2

Reply:
What about the indentation? To set the same for everyone

Reply:
But the others have the same indentation. Ex https://i.postimg.cc/HsyWyvNY/Screen-Shot-2022-09-16-at-09-56-09.png